pkg/views/util/globals.go

```diff
@@ -30,6 +32,10 @@ func RenderInfoMessageBold(message string) {
 	fmt.Println(lipgloss.NewStyle().Bold(true).Padding(1, 0, 1, 1).Render(message))
 }
 
+func GetListFooter(profileName string) string {
+	return lipgloss.NewStyle().Bold(true).Render("\n\nActive profile: " + profileName)
```

Comment:
Maybe it might be "prettier" to add the same padding to the footer so it's aligned.

![Screenshot 2024-03-29 at 12 17 31](https://github.com/daytonaio/daytona/assets/26512078/823eee86-41dd-4463-b531-20a813d10036)